.github/workflows/ci.yml

```diff
@@ -1,18 +1,17 @@
-name: CI
+name: test
 
 on:
   push:
     branches:
-      - main
-      - '*x'
+      - master
+      - '*.x'
     tags:
       - '*'
   pull_request:
-    branches:
-      - main
   schedule:
     # Weekly Monday 9AM build
-    - cron: "0 9 * * 1"
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 9 * * 1'
```

Comment:
Is it OK to have jobs for multiple repos all running at the same time, e.g. Monday 6am and 9am?